docker-compose.yml

```diff
@@ -5,6 +5,7 @@ services:
     build: .
     ports:
       - "8080:8080"
+
```

Comment:
@ilkerCelimli Can we revert the change here? There shouldn't have been a need for such a modification specific to the development.